extension/content.js

```diff
@@ -220,6 +220,46 @@ function showRecentlyPushedBranches() {
 	$('.repository-content').prepend(div);
 }
 
+// Add option for viewing diffs without whitespace changes
+function addDiffViewWithoutWhitespaceOption() {
+	if ($('.diff-options-content').length < 1 && $('.btn-group .selected[href*="diff="]').length < 1) {
+		return;
+	}
+
+	// Detect if PR or commit
+	let type = 'pr';
+
+	if ($('.btn-group .selected[href*="diff="]').length > 0) {
+		type = 'commit';
+	}
+
+	const optionElement = document.createElement('a');
+	const matchUrlArgs = new RegExp(/(\?|&)(w=1)(?=&|#|$)/i);
```

Comment:
Use the new native API for this instead: https://developers.google.com/web/updates/2016/01/urlsearchparams?hl=en